.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Release Checklist
+about: All the tasks required to complete a release across languages
+title: Release v<fill in>
+labels: ''
+assignees: ''
+
+---
+
+Full release instructions are at: [RELEASE.md](https://github.com/sigstore/protobuf-specs/RELEASE.md)
+
+## Pre Release
+- [ ] Check mediatype version of [Bundle](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto)
+- [ ] Check mediatype version of [TrustedRoot](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_trustroot.proto),
+- [ ] Update [pyproject.toml](gen/pb-python/pyproject.toml) so the `version` matches the targeted release.
+
+## Releases Complete
+- [ ] Java
+- [ ] Go
+- [ ] Python
+- [ ] Node.js
```

Comment:
Do we do node releases yet?

Reply:
Oh I dunno, I was just throwing things in here to start getting comments.

Reply:
There will be a node release at some point. Let me see if I can track down some of the npm folks for that. But for now ignore it